### PR TITLE
Transitions to checkout screen

### PR DESCRIPTION
**Pull request by eoji**
- Adding a fake extended fab. Will create a proper style laterrr.
- Created circular reveal transition to Checkout screen.
- Love2Refactor

device-2019-02-22-170852 2019-02-22 17_09_21

### CODE REVIEW

File: app/src/main/java/com/example/izzyoji/checkoutpoc/models/Reward.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.example.izzyoji.checkoutpoc.models
+
+data class Reward(var name: String, var price: String, var description: String) {
```
Comment (Rcureton) [inline app/src/main/java/com/example/izzyoji/checkoutpoc/models/Reward.kt]: 👍🏾

File: app/src/main/java/com/example/izzyoji/checkoutpoc/CheckoutActivity.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        overridePendingTransition(0, 0);
+        setContentView(R.layout.activity_checkout)
+
+        if (savedInstanceState == null) {
+            root.visibility = View.INVISIBLE
+
+            val viewTreeObserver = root.viewTreeObserver
```
Comment (Rcureton) [inline app/src/main/java/com/example/izzyoji/checkoutpoc/CheckoutActivity.kt]: I wonder if we could try this [ktx extension](https://developer.android.com/kotlin/ktx#kotlin) out here.

Comment (eoji) [reply app/src/main/java/com/example/izzyoji/checkoutpoc/CheckoutActivity.kt]: Heck yeah

Comment (eoji) [reply app/src/main/java/com/example/izzyoji/checkoutpoc/CheckoutActivity.kt]: screen shot 2019-02-22 at 6 22 12 pm

Tried it but then I couldn't remove the right victim. Annoying

Comment (Rcureton) [reply app/src/main/java/com/example/izzyoji/checkoutpoc/CheckoutActivity.kt]: It's all good it was worth the try!

**Pull request merged**